pretyping/classops.ml

```diff
@@ -297,31 +292,25 @@ let lookup_path_to_fun_from env sigma s =
 let lookup_path_to_sort_from env sigma s =
   apply_on_class_of env sigma s lookup_path_to_sort_from_class
 
+let mkNamed = function
```

Comment:
This one should really be somewhere

Reply:
No, because it is disregarding the universe variables. We shouldn't make users believe that this is fine.

Reply:
@ppedrot I don't want to do things that are not considered "fine". So what should we do here, instead?

Reply:
If you really desire to call a combinator (even though there is no reason for that here) I'd use `UnivGen.fresh_global_instance` or `Evd.fresh_global`.

Reply:
Ok, I don't think it would bring anything.